include/tracing_impl/lttng.h

```diff
@@ -2,6 +2,11 @@
  * Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All rights reserved.
```

Comment:
Why the revert as part of this PR?

Reply:
Local builds were failing for me without this.

Reply:
ok, but let's sort that out in its own PR

Reply:
https://github.com/aws/aws-ofi-nccl/pull/411/commits